toc/rfc/rfc-0041-provide-custom-stacks-functionality.md

```diff
@@ -0,0 +1,362 @@
+# Meta
+
+- Name: Enhance handling of stacks in CF to improve stack removal/migration experience
```

Comment:
The title of this RFC fits better to RFC https://github.com/cloudfoundry/community/pull/1220. I think the titles have been swapped by mistake.